app/views/welcome/gg.haml

```diff
@@ -2,9 +2,6 @@
   %h1 TightFit
   %p fitting porn into your tight schedule
 
-  - 5.times do
-    %video{ controls: true, width: "320", height: "240" }
-      %source{ src: @videos.sample.link, type: "video/mp4" }
```

Comment:
I removed this cause it was causing my rails app to crash. Not sure if I should add it back in.

Reply:
Ya leave it out, probably something to do with heroku